tests/integration-tests/utils.py

```diff
@@ -137,6 +138,59 @@ def retrieve_cfn_outputs(stack_name, region):
     return _retrieve_cfn_data(stack_name, region, "Output")
 
 
+def create_metric(unique_name, metric_name, cluster_name, period_length_sec, stat):
+    return {
+        "Id": unique_name,
+        "MetricStat": {
+            "Metric": {
+                "Namespace": "ParallelCluster/Errors/" + cluster_name,
+                "MetricName": metric_name,
+            },
+            "Period": period_length_sec,
+            "Stat": stat,
+        },
+    }
+
+
+def retrieve_metric_data(
+    unique_name, cluster_name, metric_name, period_length_sec, collection_time_min, stat="Sum", region="us-east-1"
```

Comment:
Can you use `metric_names` and `unique_names` to indicate they are lists?

Reply:
dang it ok